src/ship.js

```diff
@@ -0,0 +1,16 @@
+const Ship = (size, name, first, orientation) => ({
+  size,
+  name,
+  first,
+  orientation,
+  life: size,
+  hit() { this.life -= 1; },
+  isSunk() {
+    if (this.life === 0) {
+      return true;
+    }
+    return false;
+  },
+});
+
+module.exports = Ship;
```

Comment:
We need to use ES6, export, import, syntax, as stated in the [Pathwright to this project](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622118/)
![image](https://user-images.githubusercontent.com/46115725/73208810-b2005280-410c-11ea-8801-b9168287bdcf.png)